pkg/apis/migration/v1alpha1/template.go

```diff
@@ -0,0 +1,60 @@
+package v1alpha1
+
+import (
+	"strings"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+type TemplateResources struct {
+	Templates []TemplateResource `json:"templates,omitempty"`
+}
+
+type TemplateResource struct {
+	Resource     string `json:"resource"`
+	TemplatePath string `json:"templatePath"`
+}
+
+var DefaultTemplates = []TemplateResource{
+	{
+		Resource:     "cronjob.batch",
+		TemplatePath: ".spec.jobTemplate.spec.template",
+	},
```

Comment:
I think job (job.batch) should be included here too.

Reply:
Thanks! Missed this part.